cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/transform/ScriptFilterTransform.java

```diff
@@ -98,6 +102,10 @@ private void init() {
     ScriptEngineManager manager = new ScriptEngineManager();
     engine = manager.getEngineByName("JavaScript");
 
+    // initialize metrics and logging
+    engine.put("metrics", metrics);
```

Comment:
should describe how to use these in the docs, otherwise nobody will know they exist.